.github/workflows/deploy.template.yml

```diff
@@ -51,19 +51,40 @@ jobs:
       - name: Install package
         run: poetry install
```

Comment:
Flagging that I got a `PermissionError` here on my test innovate_tagging project. I side-stepped it by commenting out `poetry config virtualenvs.create false` on the "Install and configure poetry" task